pennylane/drawer/__init__.py

```diff
@@ -14,15 +14,14 @@
 """
 This module provides the circuit drawing functionality used to display circuits visually.
 
-.. currentmodule:: pennylane.circuit_drawer
+.. currentmodule:: pennylane.drawer
 .. autosummary::
     :toctree: api
 
 """
 
+from .draw import draw, draw_mpl
 from .tape_text import tape_text
 from .tape_mpl import tape_mpl
 from .mpldrawer import MPLDrawer
-from .drawable_layers import drawable_layers
-from .utils import convert_wire_order
```

Comment:
These two methods are only accessed internally by `tape_text` and `tape_mpl`.  I don't see a reason we need to make them accessible from here.